packages/service-library/src/servicelib/zip_stream/_constants.py

```diff
@@ -0,0 +1,8 @@
+from typing import Final
+
+from pydantic import ByteSize, TypeAdapter
+
+DEFAULT_READ_CHUNK_SIZE: Final[int] = TypeAdapter(ByteSize).validate_python("1MiB")
+MIN_MULTIPART_UPLOAD_CHUNK_SIZE: Final[int] = TypeAdapter(ByteSize).validate_python(
```

Comment:
we already have such a constant in https://github.com/ITISFoundation/osparc-simcore/blob/5732a12e07e63d5ce55010ede9b9ab543bb9b278/packages/aws-library/src/aws_library/s3/_constants.py any reason why you do not use this?